packages/cli/yaml/docs-validator/src/rules/valid-file-types/valid-file-types.ts

```diff
@@ -27,16 +28,20 @@ export const ValidFileTypes: Rule = {
     create: () => {
         return {
             filepath: async ({ absoluteFilepath, value }) => {
-                const file = await readFile(absoluteFilepath);
-                const isValid = await isValidFileType(file);
-
-                if (!isValid) {
-                    return [
-                        {
-                            severity: "error",
-                            message: `File type of ${value} is invalid`
-                        }
-                    ];
+                const doesExist = await doesPathExist(absoluteFilepath);
+
+                if (doesExist) {
+                    const file = await readFile(absoluteFilepath);
+                    const isValid = await isValidFileType(file);
+
+                    if (!isValid) {
+                        return [
+                            {
+                                severity: "error",
+                                message: `File type of ${value} is invalid`
+                            }
+                        ];
+                    }
                 }
```

Comment:
should we add a warning or error if the file doesn't exist?

Reply:
there's another rule that checks that!